tensorflow/python/keras/optimizer_v2/optimizer_v2.py

```diff
@@ -138,15 +138,13 @@ class directly, but instead instantiate one of its subclasses such as
     loss = <call_loss_function>
   vars = <list_of_variables>
   grads = tape.gradient(loss, vars)
-  processed_grads = [process_gradient(g) for g in grads]
-  grads_and_vars = zip(processed_grads, var_list)
 
-  # grads_and_vars is a list of tuples (gradient, variable).  Do whatever you
-  # need to the 'gradient' part, for example cap them, etc.
-  capped_grads_and_vars = [(MyCapper(gv[0]), gv[1]) for gv in grads_and_vars]
+  # Do whatever you need to the gradients, for example cap them, etc.
```

Comment:
`Apply pre-processing` or `Process` instead of `Do whatever you need`

Reply:
Sure. How do I change it?